### ISSUE TITLE
Show channel funding tx in tx history

### ISSUE THREAD

**Issue by bonomat** [get10101/10101#1800]
I'm working on displaying the balance at the moment and things don't add up

For example: 

- user initial on-chain balance: 100,000,000
- order price: 40,000
- leverage: 2
-->
- margin: 12500
- fee: 75

We show this correctly in the tile for opening a position: 

<img width="318" alt="image" src="https://github.com/get10101/10101/assets/224613/15f37585-629e-4bfb-90d0-9e2386390f6f">

But the outgoing transaction shows a different value:

<img width="320" alt="image" src="https://github.com/get10101/10101/assets/224613/d41ae3cd-c737-4a4c-b4a5-09568fdff660">

- outgoing tx: 16,904
- on-chain fee: 4,698 (already included in 16,904)

So basically we fund the dlc-channel with `16,904-4,698`=`12,206`?

Related to https://github.com/get10101/10101/issues/1799

**Comment by bonomat** [comment]
My conclusion is that the amounts are actually correct. We have funded the channel with 12500 and the remaining amount is for the transaction fees reserved for buffer/refund and cets. 

https://github.com/p2pderivatives/rust-dlc/blob/adc7cef5650cbba56f869a275b7b0a273c414ef1/dlc/src/lib.rs#L277-L328

I'll see to simply update the tile.
